src/Sidekick.Common.Ui/Styles/main.css

```diff
@@ -1,14 +1,56 @@
-@import "flowbite/src/themes/default";
+@import 'flowbite/src/themes/default' layer(base);
 
-@import "./fonts.css";
-@import "./item-headers.css";
-@import "./item-logbook-factions.css";
-@import "./item-sockets.css";
-@import "./poe.css";
+@import './fonts.css' layer(base);
+@import './item-headers.css' layer(base);
+@import './item-logbook-factions.css' layer(base);
+@import './item-sockets.css' layer(base);
+@import './poe.css' layer(base);
 
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
```

Comment:
As we control the source of the files we want tailwind to parse, we can make sure tailwind doesn't include any unwanted source.
```
@import "tailwindcss" source(none);
```